engine/webview.go

```diff
@@ -57,6 +57,10 @@ func (e *WebviewEngine) Bind(name string, fn app.Func) error {
 	})
 }
 
+func (e *WebviewEngine) Eval(js string) {
+	e.webview.eval(js)
```

Comment:
Can you change it to `Eval` with uppercase `E`

Reply:
sure, done.